migrations/v2.js

```diff
@@ -0,0 +1,531 @@
+        },
+        _logging: logging
+      }
+    ]
+  });
+
+  testSuccessWhere('empty course/config', {
+    content: [
+      { _type: 'course' },
+      { _type: 'config' }
+    ]
+  });
+
+  testStopWhere('no course/config', {
+    content: [{ _type: 'other' }]
+  });
+});
+
+describe('core - update to v2.0.18', async () => {
+  let course;
+
+  whereContent('core - where course', async (content) => {
+    course = getCourse();
+    return course;
+  });
+
+  mutateContent('core - update _globals._accessibility._accessibilityToggleTextOn', async (content) => {
+    const originalValue = _.get(course, '_globals._accessibility._accessibilityToggleTextOn');
+    _.set(course, '_globals._accessibility.accessibilityToggleTextOn', originalValue || 'Turn accessibility on?');
+    _.unset(course, '_globals._accessibility._accessibilityToggleTextOn');
+    return true;
+  });
+
+  mutateContent('core - update _globals._accessibility._accessibilityToggleTextOff', async (content) => {
+    const originalValue = _.get(course, '_globals._accessibility._accessibilityToggleTextOff');
+    _.set(course, '_globals._accessibility.accessibilityToggleTextOn', originalValue || 'Turn accessibility off?');
+    _.unset(course, '_globals._accessibility._accessibilityToggleTextOff');
+    return true;
+  });
+
+  checkContent('core - check _globals._accessibility.accessibilityToggleTextOn', async (content) => {
+    const isValid = _.has(course, '_globals._accessibility.accessibilityToggleTextOn');
+    if (!isValid) throw new Error('core - _globals._accessibility.accessibilityToggleTextOn not added');
+    return true;
+  });
+
+  checkContent('core - check _globals._accessibility.accessibilityToggleTextOff', async (content) => {
+    const isValid = _.has(course, '_globals._accessibility.accessibilityToggleTextOff');
+    if (!isValid) throw new Error('core - _globals._accessibility.accessibilityToggleTextOff not added');
+    return true;
+  });
+
+  testSuccessWhere('course with globals', {
+    content: [
+      {
+        _type: 'course',
+        _globals: {
+          _accessibility: {
+            _accessibilityToggleTextOn: 'Turn accessibility on?',
+            _accessibilityToggleTextOff: 'Turn accessibility off?'
+          }
+        }
+      }
+    ]
+  });
+
+  testSuccessWhere('course with no globals', {
+    content: [
+      { _type: 'course' }
+    ]
+  });
+
+  testStopWhere('no contentObjects', {
+    content: [{ _type: 'config' }]
+  });
+});
+
+describe('core - update to v2.1.0', async () => {
+  let config;
+
+  whereContent('core - where config', async (content) => {
+    config = getConfig();
+    return config;
+  });
+
+  mutateContent('core - update _accessibility._isEnabledOnTouchDevices', async (content) => {
+    const originalValue = _.get(config, '_accessibility._isEnabledOnTouchDevices');
+    _.set(config, '_accessibility.accessibilityToggleTextOn', originalValue || 'Turn accessibility on?');
```

Comment:
Is this right? It doesn't seem right in the context of the mutation title but it does seem to match the checkContent.